aten/src/ATen/Dispatch.h

```diff
@@ -280,6 +280,25 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
     }                                                                       \
   }()
 
+#define AT_DISPATCH_FLOATING_AND_QINT_TYPES(TYPE, NAME, ...)          \
```

Comment:
oh I thought we can compose...would be even better if we can compose AT_DISPATCH_FLOATING_TYPES and AT_DISPATCH_QINT_TYPES somehow.

Reply:
e.g. moving float cases and int cases to a macro

Reply:
Composition would require rewriting the dispatch switches to avoid the the default case.

Reply:
can we just define a macro for 3 AT_QINT_PRIVATE_CASE_TYPE and 2 AT_PRIVATE_CASE_TYPE ?
e.g.
```
#define QINT_CASES(__VA_ARGS__) AT_QINT_PRIVATE_CASE_TYPE(...) \
   AT_QINT_PRIVATE_CASE_TYPE(...) \
   AT_QINT_PRIVATE_CASE_TYPE(...)
```